content/en/platform/corda/5.0-alpha/deploying/deployment-tutorials/cordapp-packaging.md

```diff
@@ -79,18 +79,24 @@ This key can be generated once and kept for reuse.
 
 ## Generating a Group Policy File
 
-To generate a group policy file:
+If you intend to run a basic [static network](../network-types.html#static-networks) setup, you can use the Corda CLI to generate a group policy file. For example, you could run the following command to generate the file:
 ```shell
 ./corda-cli.sh mgm groupPolicy > GroupPolicy.json
 ```
-For more information about `mgm` commmands, see [CLI commands](../corda-cli/commands.html#mgm).
+
+Refer to the [Group Policy](../../../../../../en/platform/corda/5.0-alpha/deploying/group-policy.html) section to learn more about the `GroupPolicy.json` file.
+
+<!-- For more information on this plugin, refer to the [README.md](https://github.com/corda/corda-runtime-os/blob/release/os/5.0/tools/plugins/mgm/README.md). -->
+
+If you are running a dynamic network, you will need to export the group policy file from the MGM using the HTTP API. To learn how to do this, see the [MGM Onboarding](../deployment-tutorials/onboarding/mgm-onboarding.html) tutorial for more information.
+
 ## Building a CPI
 
-The gradle plugin builds the CPB. To create a CPB for a CPI:
-```shell
+The Gradle plugin builds the CPB. Run the following command to turn a CPB into a CPI:
+```shell 
 ./corda-cli.sh package create-cpi \
     --cpb mycpb.cpb \
-    --group-policy GroupPolicy.json \
+    --group-policy TestGroupPolicy.json \
```

Comment:
The filename should match the export command on line 84. So GroupPolicy.json was correct.